server/models/Request.js

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose');
+
+const requestSchema = new mongoose.Schema({
+  user_id: {
+    type: String,
+  },
+  sitter_id: {
+    type: String,
+  },
+  start: {
+    type: Number,
+  },
+  end: {
+    type: Number,
+  },
```

Comment:
These ones should be of type `Date` with a default value. Please refer to the issue.

Reply:
These changes have now been made and approved in https://github.com/hatchways/team-igneous/pull/56/commits/8ade7c03ebe6f587d3dc02b8f8aa85f5e9dcf490 :)